docs/concepts/architecture/master-node-communication.md

```diff
@@ -106,3 +110,79 @@ cluster (connecting to the ssh server listening on port 22) and passes all
 traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the private
 GCE network in which the cluster is running.
+
+
+
+## Configuring communication ports
+
+> A note on the difference between specifying 0.0.0.0 or 127.0.0.1 as a bind `--address`. *0.0.0.0* will bind to ALL interfaces on the host. *127.0.0.1* will ONLY bind to localhost.
+
+
+### kube-apiserver
+
+| Argument                       | Default           | Description                              |
+| ------------------------------ | ----------------- | ---------------------------------------- |
+| --advertise-address            | --bind-address    | The IP address on which to advertise the apiserver to members of the cluster. This address must be reachable by the rest of the cluster |
+| --bind-address                 | default interface | The IP address on which to listen for the --secure-port port. The associated interface(s) must be reachable by the rest of the cluster, and by CLI/web clients. |
+| --etcd-servers                 |                   | List of etcd servers to connect with (scheme://ip:port), comma separated. |
+| --insecure-bind-address        | 127.0.0.1         | The IP address on which to serve the --insecure-port |
+| --insecure-port                | 8080              | The port on which to serve unsecured, unauthenticated access. It is assumed that firewall rules are set up such that this port is not reachable from outside of the cluster and that port 443 on the cluster's public address is proxied to this port. This is performed by nginx in the default setup. |
+| --kubernetes-service-node-port |                   | If non-zero, the Kubernetes master service (which apiserver creates/maintains) will be of type NodePort, using this as the value of the port. If zero, the Kubernetes master service will be of type ClusterIP. |
+| --secure-port                  | 6443              | The port on which to serve HTTPS with authentication and authorization. Set to 0 to disable |
+| --service-node-port-range      | 30000-32767       | A port range to reserve for services with NodePort visibility. |
+
+
+| Argument     | Default                     | Description                              |
```

Comment:
Why are these two flags here?

Reply:
It is missing a header (E.g Specifying the location / authorization to the master)  - these are common across most of the services (except kubelet, hence the note that --master not applying).

Does it make more sense to duplicate this across all the different services? 
I also noticed that the flags are being deprecated in favor of a config file - should I maybe not switch to a config file based documentation with a note on how to map back to the deprecated arguments?

Reply:
@mtaufen @ncdc WDYT?

Reply:
We should encourage people to use the config file, but there's no harm in also telling them how to do things with flags, as long as we are clear that the flags are eventually going away.